src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -121,4 +122,83 @@ public static Tag parseTag(String tag) throws ParseException {
         }
         return tagSet;
     }
+
+    /**
+     * Parses a {@code String name} into a {@code DateApplied}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code name} is invalid.
+     */
+    public static String parseDateApplied(String dateapplied) throws ParseException {
+        requireNonNull(dateapplied);
+        String trimmedDate = dateapplied.trim();
+        // Include bottom when DateApplied class is merged
+        //if (!Name.isValidName(trimmedDate)) {
+        //throw new ParseException(DateApplied.MESSAGE_CONSTRAINTS);
+        //}
+        return new String(trimmedDate);
+    }
+
+    /**
+     * Parses a {@code String name} into a {@code DateInterview}.
```

Comment:
Should change {String name} to {String dateInterview}. Also can return InterviewDate(trimmedDate) since there is InterviewDate class.

Reply:
Parser will be returning the argument string that will be used to create an InterviewDate so it should return a string